app/views/home/form_name.html.slim

```diff
@@ -1,23 +1,29 @@
+= render('shared/error_block', form: @form_name) if @form_name.errors.any?
 h2.heading-large
-  span.heading-secondary Question 9 of 14
-  | What is the form name or number related to this application?
+  span.heading-secondary =t('breadcrumb', scope: @form_name.i18n_scope)
+  =t('text', scope: @form_name.i18n_scope)
 
-p eg ‘N1’ or ‘notice to appeal’ form
+p =t('example', scope: @form_name.i18n_scope)
 
 
 = form_for @form_name, url: form_name_save_path do |f|
-  .form-group
-    = f.text_field :identifier, class: 'form-control'
+  fieldset
+    legend.visuallyhidden =t('text', scope: @form_name.i18n_scope)
+    .form-group  class=('error' if @form_name.errors.any?)
+      label.form-label for='form_name_identifier'
+        span.visuallyhidden =t('prompt', scope: @form_name.i18n_scope)
+        span.error-message#paid = @form_name.errors[:identifier].join(' ') if @form_name.errors[:identifier].any?
```

Comment:
#paid needs to be the field name (#identifier), so that the error block at the top of the page links to the right field